localtypings/pxtarget.d.ts

```diff
@@ -146,6 +146,7 @@ declare namespace pxt {
         simAnimationEnter?: string;
         simAnimationExit?: string;
         projectGallery?: string;
+        gettingStarted?: string;
```

Comment:
I thought the sideDocs property enabled/disabled showing content in the side docs. 
My thoughts on the gettingStarted property is to be a flag to enable / disable whether or not the button shows up.